ubuntu/usb/mtp-server.conf

```diff
@@ -0,0 +1,27 @@
+description "MTP protocol server"
```

Comment:
You can remove this, that file is in an identical version already in the root filesystem.

Reply:
removed